src/dotnet/Users.Service/Controllers/MobileAuthController.cs

```diff
@@ -272,4 +322,26 @@ private async Task WriteAutoClosingMessage(CancellationToken cancellationToken)
         HttpContext.Response.ContentType = "text/html; charset=utf-8";
         await HttpContext.Response.WriteAsync(responseString, cancellationToken: cancellationToken).ConfigureAwait(false);
     }
+
+    private async Task AuthenticateSessionWithPrincipal(
+        string sessionId,
+        ClaimsPrincipal principal,
+        CancellationToken cancellationToken)
+    {
+        var oldUser = HttpContext.User;
+        HttpContext.User = principal;
+        try
+        {
+            var helper = Services.GetRequiredService<ServerAuthHelper>();
+            await helper.UpdateAuthState(
+                    new Session(sessionId),
+                    HttpContext,
+                    cancellationToken)
+                .ConfigureAwait(false);
+        }
+        finally
+        {
```

Comment:
Backet should be the same line as finally

Reply:
fixed